plugins/rtmp-services/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 project(rtmp-services)
 
 option(ENABLE_SERVICE_UPDATES "Checks for service updates" OFF)
+if(NOT ENABLE_SERVICE_UPDATES)
```

Comment:
Change the condition to apply to Windows only.

Reply:
So far there is no status output in the affirmative, so I don't see why there needs to be an output _just_ for the negative case.